docs/docs/zh/guide/parameter/context.md

```diff
@@ -76,3 +77,22 @@ Node_mysql 运行结果如下：
 虽然在 Node_A 的脚本中为 output 赋值为 1，但日志中显示的值仍然为 100。但根据[参数优先级](priority.md)的原则：`本地参数 > 上游任务传递的参数 > 全局参数`，在 Node_B 中输出的值为 1。则证明 output 参数参照预期的值在该工作流中传递，并在 Node_mysql 中使用该值完成查询操作。
 
 但是 value 的值却只有在 Node_A 中输出为 66，其原因为 value 的方向选择为 IN，只有当方向为 OUT 时才会被定义为变量输出。
+
+
+### HTTP
+
+拖一个http类型的任务，在自定义参数中KEY的位置填写body，在IN/OUT的部分选择OUT，在输出数据类型部分选择VARCHAR，一定要选择VARCHAR，不要选择别的哈。
```

Comment:
Documentation should avoid colloquialism.
```suggestion
拖一个http类型的任务，在自定义参数中KEY的位置填写body，在IN/OUT的部分选择OUT，在输出数据类型部分选择VARCHAR（只能选择VARCHAR）。
```

Reply:
It has been processed and the code has been committed